README.md

```diff
@@ -74,6 +74,16 @@ the meantime, you can use
 [brianloveswords/base64url](https://github.com/brianloveswords/base64url)
 to decode the signature.
 
+As of nodejs `v0.11.8`, SPKAC support was introduce. If your node
```

Comment:
(nit) I think the verison number would be better represented as just emphasized, as it's not code, *v_0.11.8*